### PR TITLE
[ENTVTX-110] Don't hardcode the route name in test

### PR DESCRIPTION
**Pull request by akoniar**
Please review
@Ladicek
@spisiakm

### CODE REVIEW

File: src/test/java/io/openshift/booster/OpenShiftIT.java [hunk 1]
```diff
@@ -26,15 +26,15 @@
 public class OpenShiftIT {
     private String project;
 
-    private final String applicationName = "http-vertx";
+    private static final String APPLICATION_NAME = System.getProperty("app.name","http-vertx");
```
Comment (spisiakm) [inline src/test/java/io/openshift/booster/OpenShiftIT.java]: @akoniar could you please roll back the changes here so we can keep the boosters in sync? Thanks!

Comment (Ladicek) [reply src/test/java/io/openshift/booster/OpenShiftIT.java]: Strictly speaking they aren't totally in sync anyway, as @akoniar used `application.name` and I used `app.name`.

Comment (Ladicek) [reply src/test/java/io/openshift/booster/OpenShiftIT.java]: Strictly speaking they aren't totally in sync anyway, as @akoniar used `application.name` and I used `app.name`.

Comment (spisiakm) [reply src/test/java/io/openshift/booster/OpenShiftIT.java]: Yeah, I just noticed this difference is in a few other boosters as well. I've already merged some PRs though.

**Comment by spisiakm** [comment]
Thanks @akoniar :)

**Pull request merged**